src/runtime/js/ts.ui/core/core-gui@tradeshift.com/spirits/ts.ui.Spirit.js

```diff
@@ -51,6 +51,17 @@ ts.ui.Spirit = (function using(Type, GuiArray, confirmed, chained) {
 				this.busy(false);
 			}),
 
+			/**
+			 * Change message.
+			 * @param @optional {string} message
+			 * @returns {this}
+			 */
+			spin: chained(function(message) {
```

Comment:
I am not a fan of this method name 🤔 Can we use more expressive name for it? What do you think ?

Reply:
I used spinText before. But also not good. I don't want to use message(), text(). Seems to general. It should be only for the spinner. What do you think, Any good ideas?

Reply:
I see 🤔 How about `spinnerMessage`?